classic/com.limelight.json

```diff
@@ -14,6 +14,16 @@
     ],
     "releases": [
         {
+            "name": "8.11",
+            "versionCode": 216,
+            "uuid": "eddb769c-1e16-4a22-91e3-5a3c60c503fd",
+            "date": "2020-03-25T00:00:00Z",
+            "url": "https://github.com/moonlight-stream/moonlight-android/releases/download/v8.11/app-nonRoot-release.apk",
+            "size": 5397046,
+            "md5sum": "d23cf9469f4055e511474f9554a2e4d6",
+            "latest": true
+        },
+		{
```

Comment:
indention here is wrong.
You also need to set "latest" to false in the older release.